### PR TITLE
feat: Updates inputs to use labels instead of placeholders

### PR DESCRIPTION
**Pull request by manueltanzi-okta**
### Description

- Placeholders for inputs resulted to be not good in term of accessibility. Converting to use top labels.
- Also removing icons inside inputs.

JIRA: https://oktainc.atlassian.net/browse/OKTA-210951

**Note**: Will go in okta-signin-widget v3.0.0.

### Screenshots

Screen Shot 2019-05-10 at 10 56 35 AM

Screen Shot 2019-05-10 at 10 56 01 AM

Screen Shot 2019-05-10 at 10 56 21 AM

InlineTOTPForm
TOTPForm
EnterPasscodeForm
IDPDiscoveryForm
UnlockAccountController
EnrollCallAndSmsController (Voice)
EnrollOnPremController2
EnrollOnPremController
EnrollSymantecVipController
EnrollCallAndSmsController2
EnrollCallAndSmsController1
EnrollCallAndSmsController
PasswordExpiredController
PasswordResetController

### Reviewers

@hor-kanchan-okta @haishengwu-okta @alexdahl-okta @nbarbettini

### CODE REVIEW

**Comment by nbarbettini** [comment]
cc @maggielaw-okta 

Awesome work @manueltanzi-okta! 🎉 

Can we make the labels bolder to match the mock?

File: src/UnlockAccountController.js [hunk 1]
```diff
@@ -99,14 +99,15 @@ function (Okta, FormController, Enums, FormType, ValidationUtil, ContactSupport,
         }
         else {
           formChildren.push(FormType.Input({
-            placeholder: Okta.loc('account.unlock.email.or.username.placeholder', 'login'),
+            label: Okta.loc('account.unlock.email.or.username.placeholder', 'login'),
+            'label-top': true,
             name: 'username',
             input: TextBox,
+            inputId: 'account-recovery-username',
```
Comment (haishengwu-okta) [inline src/UnlockAccountController.js]: curious why need to hard code inputId?

**Comment by maggielaw-okta** [comment]
+1 to making labels match design. Thx!

File: src/UnlockAccountController.js (hunk 1 above)
Comment (manueltanzi-okta) [reply src/UnlockAccountController.js]: To have a handy selector for testing. Without hard coding that, id will be generated, example: "input66"

**Comment by maggielaw-okta** [comment]
Looks great!

**Comment by alexdahl-okta** [comment]
lgtm! 🎉

**Comment by OkGoDoIt** [comment]
Awesome step forward for usability.  Is there a reason we are removing the icons though?

**Comment by maggielaw-okta** [comment]
We decided the icons weren't paying enough rent (mostly decorative, not functional), so it's an effort to de-clutter. We're gettin' our KanMari method on. :)

**Comment by nbarbettini** [comment]
> Do the (?) on the right become redundant, since they have the same label?

The tooltip text is identical to the label by default, but you can customize it. Some developers use this to add extra hints or instructions. Longer term (i.e. not in this PR) we plan to improve the design and accessibility of these tooltips/hints, too.

**Pull request merged**